adobe_vipm/flows/utils/validation.py

```diff
@@ -107,3 +112,33 @@ def is_reseller_change(order: dict) -> bool:
     return agreement_type == "Transfer" and is_ordering_param_required(
         order, Param.CHANGE_RESELLER_CODE
     )
+
+
+def validate_government_lga_data(order: dict, adobe_data: dict):
+    """
+    Validates the government transfer.
+
+    Args:
+        order: MPT order
+        adobe_data: Adobe customer data
+        is_preview: Flag indicating whether validation is a preview.
+
+    Returns:
+        True if the government transfer is valid, False otherwise.
+    """
```

Comment:
We're missing the Raises docs

Raises:
    GovernmentLGANotValidOrderError: ...
    GovernmentNotValidOrderError: ....